backend/inspirehep/pidstore/minters/doi.py

```diff
@@ -12,3 +12,10 @@
 class DoiMinter(Minter):
     pid_value_path = "dois.value"
     pid_type = "doi"
+
+    def create(self, pid_value):
+        super().create(pid_value.lower())
+
+    def get_pid_values(self):
+        pid_values = [str(pid_value).lower() for pid_value in super().get_pid_values()]
+        return set(pid_values)
```

Comment:
You can do a set comprehension directly:
```python
return {str(pid_value).lower() for pid_value in super.().get_pid_values()}
```

Reply:
:+1: